### PR TITLE
Feature/small tasks

### PR DESCRIPTION
**Pull request by hellangle**

### CODE REVIEW

File: Code/Jarboo.Admin.Web/Controllers/AccountsController.cs [hunk 1]
```diff
@@ -36,7 +36,7 @@ public partial class AccountsController : BaseController
         public virtual ActionResult Login(string returnUrl = "")
         {
             ViewBag.ReturnUrl = returnUrl;
-            return View(new LoginViewModel());
+            return View(new LoginViewModel() { Persist = true });
```
Comment (vgoloviznin) [inline Code/Jarboo.Admin.Web/Controllers/AccountsController.cs]: Why this line was added?

File: Code/Jarboo.Admin.Web/Infrastructure/Configuration.cs [hunk 2]
```diff
@@ -10,12 +10,12 @@
 
 namespace Jarboo.Admin.Web.Infrastructure
 {
-    public class Configuration : IGoogleDriveConfiguration
+    public class Configuration : IGoogleDriveConfiguration, IMandrillConfiguration
```
Comment (vgoloviznin) [inline Code/Jarboo.Admin.Web/Infrastructure/Configuration.cs]: Please, use the new settings logic which relies in the DB and not on config files

File: Code/Jarboo.Admin.Web/Views/Shared/EditorTemplates/DataTableTaskFilter.cshtml [hunk 3]
```diff
@@ -13,30 +17,28 @@
...
 
     <div class="checkbox">
         <label>
             <input type="checkbox" class="checkbox" name="@Html.NameFor(x => x.ShowDone)"
-                @if (Model.ShowDone)
-                {
-                    @: checked="checked"
-                }
-            />
+                   @if (Model.ShowDone) { @: checked="checked"
```
Comment (vgoloviznin) [inline Code/Jarboo.Admin.Web/Views/Shared/EditorTemplates/DataTableTaskFilter.cshtml]: Why can't you simply use @Html.CheckboxFor(x => x.ShowDone) here?

File: Code/Jarboo.Admin.DAL/Entities/Setting.cs [hunk 4]
```diff
@@ -21,6 +21,7 @@ public class Setting : BaseEntity
         public string MandrillFrom { get; set; }
         public string MandrillPasswordRecoveryTemplate { get; set; }
         public string MandrillNewTaskTemplate { get; set; }
+        public string MandrillNewEmployeeTemplate { get; set; }
```
Comment (vgoloviznin) [inline Code/Jarboo.Admin.DAL/Entities/Setting.cs]: You've added a new field, but you didn't add a database migration to include this field

**Comment by vgoloviznin** [comment]
Please, fix the comments stated above

File: Code/Jarboo.Admin.Web/Controllers/AccountsController.cs (hunk 1 above)
Comment (hellangle) [inline Code/Jarboo.Admin.Web/Controllers/AccountsController.cs]: We have the task required Remember always checked.

File: Code/Jarboo.Admin.Web/Infrastructure/Configuration.cs (hunk 2 above)
Comment (hellangle) [inline Code/Jarboo.Admin.Web/Infrastructure/Configuration.cs]: Ok

File: Code/Jarboo.Admin.Web/Views/Shared/EditorTemplates/DataTableTaskFilter.cshtml (hunk 3 above)
Comment (hellangle) [inline Code/Jarboo.Admin.Web/Views/Shared/EditorTemplates/DataTableTaskFilter.cshtml]: I will change

File: Code/Jarboo.Admin.DAL/Entities/Setting.cs (hunk 4 above)
Comment (hellangle) [inline Code/Jarboo.Admin.DAL/Entities/Setting.cs]: It also existed in the change request

**Pull request closed without merging**